pkg/cmd/attestation/verify/verify_test.go

```diff
@@ -91,7 +91,6 @@ func TestNewVerifyCmd(t *testing.T) {
 				OIDCIssuer:       verification.GitHubOIDCIssuer,
 				Owner:            "sigstore",
 				PredicateType:    verification.SLSAPredicateV1,
-				SANRegex:         "(?i)^https://github.com/sigstore/",
```

Comment:
Since `SANRegex` is not longer set using `opts.Repo` or `opts.Owner` in the `RunE` function, this value should be removed since it never be set in the tests and will fail.

These tests don't call `runVerify`, which will create the enforcement criteria where SANRegex is set. They instead call a dummy function. There are other tests that call `runVerify`.